src/System.Security.Cryptography.Cng/src/System/Security/Cryptography/RSACng.cs

```diff
@@ -48,24 +47,28 @@ public RSACng(CngKey key)
             if (key.AlgorithmGroup != CngAlgorithmGroup.Rsa)
                 throw new ArgumentException(SR.Cryptography_ArgRSAaRequiresRSAKey, "key");
 
-            _legalKeySizesValue = s_legalKeySizes;
             Key = CngAlgorithmCore.Duplicate(key);
         }
 
+        public override KeySizes[] LegalKeySizes
```

Comment:
Can you update the reference assembly?  There are some edge cases where the compiler needs to know that a method is overridden.